src/components/Display/Display.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 
-const display = () => {
+const display = (props) => {
```

Comment:
## [OPTIONAL BUT STRONGLY RECOMMENDED CHANGE]

```javascript
const display = ({ value }) => (
 <div className="display">
   <div className="display-value">
      { value }
   </div>
 </div>
);
```

Reply:
I can implement this :slightly_smiling_face: